src/js/components/DataFilters/DataFilters.js

```diff
@@ -200,7 +202,7 @@ export const DataFilters = ({
           onClickOutside={() => setShowContent(undefined)}
           onEsc={() => setShowContent(undefined)}
         >
-          <Box width={{ min: 'medium' }}>{content}</Box>
+          <Box width={theme.dataFilters?.content?.width}>{content}</Box>
```

Comment:
Similar comment to above, but maybe `theme.dataFilters.width`?

Reply:
changed